backoffice/src/index.scss

```diff
@@ -11,3 +11,11 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+.print-buttons {
```

Comment:
This can probably be removed now, along with the div containing the buttons on the single beacon views

Reply:
Ah right so we no longer want export  buttons at all on the Single beacon record views?